projects/isolate_proto/src/isolate_proto/controller.proto

```diff
@@ -252,8 +272,8 @@ message RegisterApplicationRequest {
     repeated File files = 12;
     // Source code of the app implementation
     optional string source_code = 13;
-    // Optional health check path for runners
-    optional string health_check_path = 14;
+    // Optional health check config for runners
+    optional ApplicationHealthCheckConfig health_check_config = 14;
```

Comment:
should i keep the tag number to `16` ? there was no issue when i tested it with `14`